CuratedContent/WorkingEffectivelyWithLegacyCode.md

```diff
@@ -2,23 +2,104 @@
 
 ## Feathers, Michael, 2005, ISBN-13: 978-0131177055
 
-Legacy software is very hard to modify and extend, as it often relies on obsolete software and hardware, lacks testing, and may have no devlopers familiar with its complexities.  This book gives one hope that we can use targeted, incremental unit testing and refactoring to turn legacy software into better tested, understood, and extensible software while implementing and delivering new functionality at the same time.
+Legacy software is very hard to modify and extend, as it often relies on obsolete software and hardware, lacks testing, and may have no developers familiar with its complexities.
+This book gives us hope that we can use targeted, incremental unit testing and refactoring to turn legacy software into better tested, understood, and extensible software while implementing and
+delivering new functionality at the same time.
 
-If one read the book "Refactoring" by Martin Fowler and felt a little uneasy about how this would apply to large nasty software projects, then "Working Effectively with Legacy Code" is the book to read.
-What’s more, Michael mentions and reviews many of the object-oriented principles in Agile Software Development as well as other bits of information that one would find in other great books about software development.
-One can view this book as the logical culmination of the books "Refactoring" and "Test Driven Development" (TDD); it's where the rubber meets the road combining unit testing and refactoring.
+If you've read *Refactoring*<sup>[1]</sup> and felt a little uneasy about how this would apply to large, nasty software projects, then *Working Effectively with Legacy Code*<sup>[2]</sup> is the book to read.
```

Comment:
I was taught that technical writing should use "one" and not "you" in grad school.  Is the current standard to you "you" and "us" to make to make it more personal?

Reply:
I think I recall being taught same. But its always read a little funny to me, too often *one* can be confused with the number one rather than reference to a person, etc. So, I find it a bit awkward and harder to read. It does help to foster gender neutrality sometimes and that is most often why I might use it. But, for the most part, I personally try to avoid its use. In this document, it was not consistently used either which gave me *license* to go adjust it in the 2-3 places it *was* used ;)

Reply:
I like the more personal "you" and "us" better.  I never really liked "one".  That is just what my Ph.D. adviser wanted so that is what I adopted.

Reply:
To be honest, I myself even try to avoid that writing *passive* form as much as I can. Sometimes, wording to do that is awkward enough (or I am not created enough to come up with a better workding) I just wind up going with *you* or *us* as a last resort.

Reply:
Since this is a blog, you may want to speak directly to the reader. I like the changes you've made.